changelogs/unreleased/fix-netbox-index-schema-path.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/core
```

Comment:
Or core, or lua. Don't know.

Reply:
Core is ok.

Reply:
Okay.

Reply:
The same as for another netbox issue (https://github.com/tarantool/tarantool/blob/master/changelogs/unreleased/fix-netbox-memory-corruption.md)